/**:
  ros__parameters:
    # vehicle config
    wheel_radius: 0.1        # wheel radius [m]
    tread_width: 0.36        # distance between L/R wheels [m]
    gear_ratio: 10.0    # gear reduction ratio
    max_whl_rps: 4.0    # max wheel vel [rps]

    # オドメトリノードの設定
    wheel_state_topic: "/wheel_radians"
    odom_topic: "/odom"
    odom_frame: "odom"
    base_frame: "base_link"
    left_joint_name: "left_wheel_joint"
    right_joint_name: "right_wheel_joint"
    publish_tf: true

vehicle_interface_node:
  ros__parameters: 
    # odrive config
    odrv_usb_port: "/dev/ttyACM0"   # odrive port
    odrv_baud_rate: 115200          # baud rate for odrive

    # cmd_vel remapping
    cmd_vel_topic: "/cmd_vel"           # topic for velocity command
    cmd_vel_joy_topic: "/cmd_vel_joy"   # topic for velocity command via gamepad
    mtr_output_topic: "/motor_cmd"      # output topic (motor rps)

    # motor ID (odrive axis configs)
    mtr_axis_l: 0   # odrive axis for left motor
    mtr_axis_r: 1   # odrive axis for right motor
